core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -753,9 +753,10 @@ object ConsumerGroupCommand extends Logging {
 
     private def getCommittedOffsets(groupId: String): Map[TopicPartition, OffsetAndMetadata] = {
       adminClient.listConsumerGroupOffsets(
-        groupId,
-        withTimeoutMs(new ListConsumerGroupOffsetsOptions)
-      ).partitionsToOffsetAndMetadata.get.asScala
+        Collections.singletonMap(groupId, new ListConsumerGroupOffsetsSpec),
+        withTimeoutMs(new ListConsumerGroupOffsetsOptions()))
+        .partitionsToOffsetAndMetadata(groupId)
+        .get().asScala
```

Comment:
nit: I had a hard time reading this block. I somehow prefer the previous format, I guess. I leave this up to you.